File: ParteJava/ProyectoPro3/ProyectoPro3Modelo/src/main/java/pe/edu/pucp/proyectopro3/modelo/webhooks/WebhookBokun.java
package pe.edu.pucp.proyectopro3.modelo.webhooks;

import java.util.List;

public class WebhookBokun {
    private int IDBokun;
    private String secretKey;
    private String accesKey;
    private List<WebhookLog> webhookLog;

    public WebhookBokun(int IDBokun, String secretKey, String accesKey, List<WebhookLog> webhookLog) {
        this.IDBokun = IDBokun;
        this.secretKey = secretKey;
        this.accesKey = accesKey;
        this.webhookLog = webhookLog;
    }

    public int getIDBokun() {
        return IDBokun;
    }

    public void setIDBokun(int IDBokun) {
        this.IDBokun = IDBokun;
    }

    public String getSecretKey() {
        return secretKey;
    }

    public void setSecretKey(String secretKey) {
        this.secretKey = secretKey;
    }

    public String getAccesKey() {
        return accesKey;
    }

    public void setAccesKey(String accesKey) {
        this.accesKey = accesKey;
    }

    public List<WebhookLog> getWebhookLog() {
        return webhookLog;
    }

    public void setWebhookLog(List<WebhookLog> webhookLog) {
        this.webhookLog = webhookLog;
    }
}

File: ParteC#/ProyectoProgramacion3/ProyectoProgramacion3Model/Model/webhooks/WebhookBokun.cs
namespace ProyectoProgramacion3Model.Model.webhooks;
public class WebhookBokun
{
    public int idBokun { get; set; }
    public string secretKey { get; set; } = string.Empty;
    public string accessKey { get; set; } = string.Empty;
    public List<WebhookLog> webhooklog { get; set; } = new List<WebhookLog>();
    public WebhookBokun(int idBokun, string secretKey, string accessKey, List<WebhookLog> webhooklog)
    {
        this.idBokun = idBokun;
        this.secretKey = secretKey;
        this.accessKey = accessKey;
        this.webhooklog = webhooklog;
    }
    public WebhookBokun()
    {
    }
}